screener.config.js

```diff
@@ -26,15 +26,21 @@ const screenerConfig = {
   },
 
   // screenshot every example in maximized mode
-  states: glob
-    .sync('docs/src/examples/**/*.tsx', { ignore: ['**/index.tsx', '**/*.knobs.tsx'] })
-    .map(examplePath => {
+  states: _.flatMap(
+    glob.sync('docs/src/examples/**/*.tsx', { ignore: ['**/index.tsx', '**/*.knobs.tsx'] }),
+    examplePath => {
       const { name: nameWithoutExtension, base: nameWithExtension } = path.parse(examplePath)
 
-      return {
-        url: `http://localhost:8080/maximize/${_.kebabCase(nameWithoutExtension)}`,
-        name: nameWithExtension,
-      }
+      return [
+        {
+          url: `http://localhost:8080/maximize/${_.kebabCase(nameWithoutExtension)}`,
+          name: nameWithExtension,
+        },
+        {
+          url: `http://localhost:8080/maximize/${_.kebabCase(nameWithoutExtension)}?rtl=true`,
+          name: `${nameWithExtension}-rtl`,
```

Comment:
### Planning for more names

Let's think through the cases we'll have here for naming.  I think we'll have many permutations possible.  How do you think we should represent the other themes in the name?  Such as:

ButtonExample, Teams light
ButtonExample, Teams dark
ButtonExample, Teams contrast

ButtonExample, RTL, Teams light
ButtonExample, RTL, Teams dark
ButtonExample, RTL, Teams contrast

### Explosion of images

This also raises another point, we'll have exponential explosion of examples as we move forward.  I was wondering if we should consider rendering _all_ examples on one page?  So, there would be a single Button image that rendered just the actual example files, but one after the other.

Thoughts?

Reply:
### Naming

I need to check test ID format requirements but buttomexample-rtl-teamsdark looks good to me. You rarely need the test id (only to tell what example/config produced that screenshot.

### Explosion

What is a *validation* for screener.io? Is it a single screenshot? In that case rendering all examples together would make sense for quota reasons.
The drawback is that (1) it would be more time consuming to say which example failed and (2) whenever you add an example to a component, the whole validation for that component fails.

Reply:
totally agree - although there are few slight drawbacks related to the approach of displaying several variations on the same page, at the same time the amount of benefits clearly outweight